src/CommonLib/hostfxr_utility.cpp

```diff
@@ -200,21 +201,21 @@ HOSTFXR_UTILITY::GetHostFxrParameters(
     }
 
     // Find all folders under host\\fxr\\ for version numbers.
-    hr = strHostFxrSearchExpression.Copy(struHostFxrPath);
+    hr = struHostFxrSearchExpression.Copy(struHostFxrPath);
     if (FAILED(hr))
     {
         goto Finished;
     }
 
-    hr = strHostFxrSearchExpression.Append(L"\\*");
+    hr = struHostFxrSearchExpression.Append(L"\\*");
     if (FAILED(hr))
     {
         goto Finished;
     }
 
     // As we use the logic from core-setup, we are opting to use std here.
     // TODO remove all uses of std?
-    UTILITY::FindDotNetFolders(strHostFxrSearchExpression.QueryStr(), &vVersionFolders);
+    UTILITY::FindDotNetFolders(struHostFxrSearchExpression.QueryStr(), &vVersionFolders);
 
     if (vVersionFolders.size() == 0)
```

Comment:
if we did not find the folder, is it an error?

Reply:
Yes, that means we can't find hostfxr.dll, so we can't run in process. All of these need logs once your logging PR gets in.